src/Xhgui/Profiles.php

```diff
@@ -22,6 +22,20 @@ public function __construct(MongoDb $db)
      */
     public function insert($profile)
     {
+	    $profile = $this->setProfilingId($profile);
```

Comment:
Fixed in commit: https://github.com/perftools/xhgui-collector/pull/2/commits/8e46f1019555a09323c52831f47b3a119683a3be